aws-ecs/ecs.tf

```diff
@@ -0,0 +1,116 @@
+locals {
+  app_name = "example-app"
+  port     = 8080
+}
+
+resource "aws_ecs_cluster" "example_app" {
+  name = "${local.app_name}"
+}
+
+resource "aws_ecs_service" "example_app" {
+  name            = "${local.app_name}"
+  cluster         = "${aws_ecs_cluster.example_app.id}"
+  launch_type     = "FARGATE"
+  task_definition = "${aws_ecs_task_definition.example_app.arn}"
+
+  network_configuration {
+    subnets          = "${var.subnets}"
+    security_groups  = "${list(aws_security_group.example_app.id)}"
+    assign_public_ip = true
+  }
+
+  desired_count = 1
+}
+
+resource "aws_ecs_task_definition" "example_app" {
+  cpu                      = "256"
+  execution_role_arn       = "${data.aws_iam_role.ecs_execution_role.arn}"
+  task_role_arn            = "${aws_iam_role.example_app.arn}"
+  family                   = "${local.app_name}"
+  memory                   = "512"
+  requires_compatibilities = ["FARGATE"]
+  network_mode             = "awsvpc"
+  container_definitions    = "${jsonencode(local.container_definitions)}"
+}
+
+locals {
+  container_definitions = [
+    {
+      name  = "${local.app_name}"
+      image = "secrethub/example-app"
```

Comment:
We can change this one to `secrethub/cli` and run `secrethub example-app serve` once it's released.

Reply:
I'd prefer to leave it `secrethub/example-app`. That makes more clear that this is just an example for demo purposes which you should replace with your own app and that it is not SecretHub functionality itself.